packages/dx-react-grid/src/plugins/filtering-state.test.jsx

```diff
@@ -186,4 +188,47 @@ describe('FilteringState', () => {
         .toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('column extensions', () => {
```

Comment:
I don't like that we inlined so many tests here. We typically move them to grid-core.

Reply:
In grid-core we are testing the `getColumnExtensionValue` function. Here we are testing that `getColumnExtensionValue` takes right arguments. They are different tests.

Reply:
We can write the only one test with check for the correct arguments supplied

Reply:
I don't think so. We should test different combination of the `columnFilteringEnabled` and `columnExtensions` props.